.github/workflows/main.yml

```diff
@@ -131,16 +139,29 @@ jobs:
           done < <(
             jq -r <<<"$VARS_JSON" 'to_entries|map("\(.key)=\(.value)\u0000")[]'
           )
-      - name: Deploy integration test stack
-        run: |
-          make deploy-it
+      - name: Get user's email address
+        id: get-email-address
+        uses: actions/github-script@v7
+        with:
+          script: |
```

Comment:
This is awesome!

I was thinking if we would want to mask this output but it looks like Github will do the `@users.noreply.github.com` if your Github settings has the email visibility set to "private", so ✅